pkg/vm/vm_test.go

```diff
@@ -91,6 +91,61 @@ func TestSimpleRun(t *testing.T) {
 
 }
 
+func TestThrow(t *testing.T) {
+
+	// Program pushes 20 to the stack
+	// exits with an error
+
+	// Push(20)
+	//THROW
+
+	builder := stack.NewBuilder()
+	builder.EmitInt(20).EmitOpcode(stack.THROW)
+
+	// Pass program to VM
+	vm := NewVM(builder.Bytes())
+
+	// Runs vm with program
+	_, err := vm.Run()
+	assert.Equal(t, "the execution of the script program end with an error", err.Error())
+
+	ctx, err := vm.InvocationStack.CurrentContext()
+	assert.Equal(t, nil, err)
+	assert.Equal(t, 1, ctx.Estack.Len())
+	assert.Equal(t, -1, ctx.Astack.Len())
+}
+
+func TestThrowIfNot(t *testing.T) {
```

Comment:
This test is the same as TestSimpleRun. Let's just rename TestSimpleRun to TestThrowIfNot

Reply:
It is not really the same. 

TestSimpleRun:
20 + 34 = 54

TestThrowIfNot:
20 + 20 = 54

Reply:
Didn't see the change in number. They are both testing the THROWIFNOT function, one should throw and other should not. I don't think we need two separate test functions for this